hazelcast/src/main/java/com/hazelcast/internal/yaml/YamlUtil.java

```diff
@@ -16,11 +16,11 @@
 
 package com.hazelcast.internal.yaml;
 
-final class YamlUtil {
+public final class YamlUtil {
```

Comment:
1. Could you add Javadoc (since it's a public class now)?
2. Would it maybe make sense to add separate unit tests for this util class, like `YamlUtilTest`?

Reply:
Done both.

Reply:
thanks!